docs/network/architecture.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Etherlink architecture
+---
+
+Etherlink's main components are its nodes and the sequencer.
+
+## Sequencer
+
+Etherlink relies on a sequencer to process transactions.
+It receives transactions from nodes and packages them into _blueprints_, which are like blocks.
+In the context of Etherlink, the transactions in a blueprint are final, but they are not final for layer 1 until they are published on layer 1.
+
+## Nodes
+
+In its life cycle, an Etherlink transaction goes through three distinct types of nodes, each responsible for different aspects of Etherlink's features.
+These three nodes are being developed as part of the [Octez software suite](https://tezos.gitlab.io/introduction/tezos.html):
+
+- EVM nodes (`octez-evm-node`): The EVM nodes maintain a local copy of the Etherlink context and expose a [JSON RPC API](https://ethereum.org/en/developers/docs/apis/json-rpc/)-compliant endpoint for clients to submit transactions to.
+They forward these transactions to the sequencer and receive blueprints that contain transactions, which they use to update their state.
+They also check Smart Rollup nodes to verify that these blueprints make it to Tezos layer 1.
+- Smart Rollup nodes (`octez-smart-rollup-node`): Smart Rollup nodes store the state of the Etherlink blockchain from the perspective of Tezos layer 1.
+
+  The sequencer sends new Etherlink blueprints to one dedicated Smart Rollup node and that node publishes those blueprints on Tezos layer 1.
```

Comment:
The use of a rollup node is an implementation detail, especially if we keep the sequencer as a black box for now

Reply:
Removed this detail.